### DISCUSSION TITLE
PPOCRv4 pre-trained model for Arabic

### DISCUSSION THREAD

**Discussion by Hegelim** [PaddlePaddle/PaddleOCR#12212] [Q&A]
- 系统环境/System Environment：
Ubuntu 18.04

- 版本号/Version：Paddle：  PaddleOCR： 
- paddleocr 2.6.1.0
- paddlepaddle-gpu 2.4.2

Since the PPOCRv4 series of models have been released, I wonder if there is an updated trained model for Arabic here https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_en/models_list_en.md#23-multilingual-recognition-modelupdating?

Btw, please help me with this related question https://github.com/PaddlePaddle/PaddleOCR/issues/10519#issue-1830059959

**Comment by SWHL** [comment]
You can refer this link: https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_en/models_list_en.md#23-multilingual-recognition-modelupdating
